packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/useMaxAmount.ts

```diff
@@ -68,27 +68,32 @@ export function useMaxAmount({
       : ethChildBalance
 
     if (!nativeCurrencyBalance) {
-      return
+      return undefined
     }
 
     const nativeCurrencyBalanceFormatted = utils.formatUnits(
       nativeCurrencyBalance,
       nativeCurrency.decimals
     )
 
+    if (
+      typeof estimatedParentChainGasFees === 'undefined' ||
+      typeof estimatedChildChainGasFees === 'undefined'
+    ) {
+      return undefined
+    }
+
     const estimatedTotalGasFees =
-      (estimatedParentChainGasFees ?? 0) + (estimatedChildChainGasFees ?? 0)
+      estimatedParentChainGasFees + estimatedChildChainGasFees
 
     const maxAmount =
       parseFloat(nativeCurrencyBalanceFormatted) - estimatedTotalGasFees * 1.4
 
-    // make sure it's always a positive number
-    // if it's negative, set it to user's balance to show insufficient for gas error
-    if (maxAmount > 0) {
-      return String(maxAmount)
+    if (maxAmount <= 0) {
+      return undefined
     }
```

Comment:
the input should still be set to the max amount available to show insufficient for gas error

otherwise clicking the max button would do nothing even though user has balance